astropy/modeling/core.py

```diff
@@ -801,6 +802,76 @@ def inverse(self):
         raise NotImplementedError("An analytical inverse transform has not "
                                   "been implemented for this model.")
 
+    @property
+    def bounding_box(cls):
+        """
+        Returns a tuple defining the bounding box limits in each input 
+        dimension, symmetric about the position, 
+        e.g. ``((y_low, y_high), (x_low, x_high))``
+
+        Input must be tuple or array-like with shape (model.n_inputs, 2) 
+        or (2 * model.n_inputs,)
+
+        ``model.bounding_box = (-5, 5, -5, 5)``
+
+        is equivalent to
+
+        ``model.bounding_box = ((-5, 5), (-5, 5))``
+
+        ``model.bbox = 'auto'`` will set the default value defined in the
+        model class.
+
+        ``model.bbox = None`` will turn off the bounding box altogether
+        """
+
+        if cls._bounding_box == 'auto':
+            return cls._bounding_box_default()
+        else:
+            return cls._bounding_box
+
+    @bounding_box.setter
+    def bounding_box(cls, limits):
+        """
+        Assigns the bounding box limits  e.g. ``((y_low, y_high), (x_low, x_high))``
```

Comment:
The bounding box docstrings need some work, e.g. I think it would be good to have one 1D, one 2D and one 3D example.

But first: what should go in the property getter and what in the property setter docstring?
Does the property setter docstring even show up in the HTML docs or should everything go in the getter docstring?
(I guess this is a question for @embray or @adonath , but @patti, if you want you can also just try it out and see what happens.)